crates/bevy_sprite/src/render/sprite.wgsl

```diff
@@ -45,7 +45,7 @@ fn fragment(in: VertexOutput) -> @location(0) vec4<f32> {
 #endif
 
 #ifdef TONEMAP_IN_SHADER
-    color = vec4<f32>(reinhard_luminance(color.rgb), color.a);
+    color = vec4<f32>(tonemapping_reinhard_luminance(color.rgb), color.a);
```

Comment:
Why not `tone_mapping`?

Also, doesn't the 2D Mesh colour material need modifying?

Reply:
Good point. Sprite, mesh 2d and color material can easily make use of this now. 
I'll leave the default as:
```tonemapping: Tonemapping::None,
deband_dither: DebandDither::Disabled,```

Per this conversation: https://discord.com/channels/691052431525675048/743663924229963868/1075131069914939523 
I won't try to make the texture bindings work for the LUTs shading for shading on the material yet. So it will be possible to use tonemapping on the material as long as it's a tonemapping option that doesn't require a LUT. We'll setup the bindings to make this work on the material with the other tonemapping options in the future.